apps/www/content/docs/installation/remix.mdx

```diff
@@ -13,47 +13,11 @@ Start by creating a new Remix project using `create-remix`:
 npx create-remix@latest my-app
 ```
 
-### Run the CLI
-
-Run the `shadcn-ui` init command to setup your project:
-
-```bash
-npx shadcn-ui@latest init
-```
-
-### Configure components.json
-
-You will be asked a few questions to configure `components.json`:
-
-```txt showLineNumbers
-Would you like to use TypeScript (recommended)? no / yes
-Which style would you like to use? › Default
-Which color would you like to use as base color? › Slate
-Where is your global CSS file? › app/tailwind.css
-Do you want to use CSS variables for colors? › no / yes
-Where is your tailwind.config.js located? › tailwind.config.js
-Configure the import alias for components: › ~/components
-Configure the import alias for utils: › ~/lib/utils
-Are you using React Server Components? › no
-```
-
-### App structure
-
-<Callout>
-
-**Note**: This app structure is only a suggestion. You can place the files wherever you want.
-
-</Callout>
-
-- Place the UI components in the `app/components/ui` folder.
-- Your own components can be placed in the `app/components` folder.
-- The `app/lib` folder contains all the utility functions. We have a `utils.ts` where we define the `cn` helper.
-- The `app/tailwind.css` file contains the global CSS.
-
 ### Install Tailwind CSS
 
 ```bash
 npm add -D tailwindcss@latest autoprefixer@latest
+npx tailwindcss init
```

Comment:
```suggestion
npx tailwindcss init --ts -p
```

[Official docs](https://tailwindcss.com/docs/guides/remix) suggests to use TypeScript by default.

Reply:
Because in the Configure components.json file, we've retained the option for the user to choose TypeScript or JavaScript, so I think we should also keep it here.

Just updated the docs like: 

If you would like to use TypeScript (recommended), you can init Tailwind CSS like this:

```bash
npx tailwindcss init --ts -p
```

Otherwise, you can init Tailwind CSS without TypeScript like this:

```bash
npx tailwindcss init -p
```